lib/meta_tags/controller_helper.rb

```diff
@@ -27,6 +28,19 @@ def render_with_meta_tags(*args, &block)
     end
     protected :render_with_meta_tags
 
+    # Processes meta tags from locales file in the name space
+    # [locale].meta_tags.[controller_name].[action].[title|description|keywords]
+    # So en.meta_tags.visitors.index.title would be loaded as title for default
+    # welcome page in the visitors controller in english.
+    def meta_tags_from_locales
+      name_space = "meta_tags.#{controller_name}.#{action_name}"
+
+      self.meta_tags[:title]       = I18n.t("#{name_space}.title") unless I18n.t("#{name_space}.title", default: '').blank?
+      self.meta_tags[:keywords]    = I18n.t("#{name_space}.keywords") unless I18n.t("#{name_space}.keywords", default: '').blank?
+      self.meta_tags[:description] = I18n.t("#{name_space}.description") unless I18n.t("#{name_space}.description", default: '').blank?
```

Comment:
Line is too long. [135/80]<br>Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.